libsolidity/ast/Types.cpp

```diff
@@ -3077,8 +3077,8 @@ vector<tuple<string, TypePointer>> FunctionType::makeStackItems() const
 		break;
 	case Kind::ArrayPush:
 	case Kind::ArrayPop:
-	case Kind::ByteArrayPush:
-		slots = {make_tuple("slot", TypeProvider::uint256())};
+		solAssert(bound(), "");
+		slots = {};
```

Comment:
```suggestion
```

Reply:
I think it's better to be explicit.

Reply:
Okay. I found it a bit confusing, since the actual member is added in line 3100. This assignment makes it seem that `slots` is `{}` for push and pop. I'm fine with keeping this.